hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -209,6 +237,83 @@ def publish_files(self, task_settings, item, publish_path):
             instances.
         :param item: Item to process
         :param publish_path: The output path to publish files to
+        """
+        # Creating a temporary file on the publish path, where the alembic from maya would be
+        # exported
+        publish_file_temp = tempfile.NamedTemporaryFile(mode='w+b',
+                                                        suffix='.abc',
+                                                        prefix='tmp',
+                                                        dir="/var/tmp/",
+                                                        delete=True)
+
+        publish_path_temp = publish_file_temp.name.replace("\\", "/")
+
+        # Deciding what type of alembic needs to be exported, based on the current plugin name
+        current_plugin = self.plugin.name
+
+        if current_plugin == "Publish Geometry":
+            # Exporting alembic to a temp location.
+            # This will later be renamed and written to the publish_path
+            self._export_abc_cache(task_settings, item, publish_path_temp)
+
+            # ...and execute it:
```

Comment:
This comment doesn't make sense here. It was earlier included as we were first framing the command, then executing it.
While modifying code, please pay attention to existing comments as well.

Reply:
addressed